tests/test_parser.py

```diff
@@ -1777,3 +1778,46 @@ def test_includes_md(md_includes):
 meatball salami beef cow venison tail ball tip pork belly.</p>
 """
     assert api.documentation[0].content.html == markdown_html
+
+
+#####
+# Test multiple methods in included external resource
+#####
+@pytest.fixture(scope="session")
+def external_resource_with_multiple_methods():
+    raml_file = os.path.join(
+        EXAMPLES + "external_resource_with_multiple_methods.raml")
+    loaded_raml_file = load_file(raml_file)
+    config = setup_config(EXAMPLES + "test-config.ini")
+    return pw.parse_raml(loaded_raml_file, config)
+
+
+def test_external_resource_with_multiple_methods(
```

Comment:
for this test and the other one - I would put two sanity checks in:

1. The # of `api.resources` is indeed what you expect. For instance, with this test, I would add `assert len(api.resources) == 6`
2. The returned methods in `/internal` are what you expect (even though you haven't touched any parser code that would affect this).